### ISSUE TITLE
Duplicate ways on splitting

### ISSUE THREAD

**Issue by tomhughes** [tomhughes/trac-tickets#1344]
**[Submitted to the original trac issue database at 10.21pm, Monday, 17th November 2008]**

There's some sort of problem with splitting/creating new ways.

At http://www.openstreetmap.org/edit?lat=52.803607&lon=-1.65954&zoom=18

I created the new bridge over the canal (involving a new node on the southern side). On one of the splits, the way stayed negative - not sure why, maybe there was a dependency. Anyway, I uploaded it and all looked fine.

But on reloading (by clicking on the Edit tab) the southern section of the road still contained the bridge segment... as did the bridge, obviously.

Hey ho.

**Comment by tomhughes** [comment]
**Author: tomevans**
**[Added to the original trac issue at 10.54am, Friday, 20th March 2009]**

I also experienced a duplicate way appearing somehow.  I haven't used potlatch very much, so am not entirely sure what I did.  I think what happened was I split a way, deleted one part of it, then extended the remaining part (I thought).  I then noticed there appeared to be two ways.  If I clicked on the older area of the way it highlighted only the old half-length.  Clicking on the newer section of the way highlighted the whole length.  So I clicked on the old section and deleted it.  This appeared to leave one way covering the entire length, as desired.

Link to the final one of the ways below.  This is the one that now still exists.  It seems to lack earlier history.

http://www.openstreetmap.org/browse/way/32284141/history

The intentionally split part of the way which I deleted is, I think, this one:

http://www.openstreetmap.org/browse/way/4234628/history

Its history implies that this was the 'original' way in the split, and I subsequently used the new way.

I can't find a trace of the other way.  Maybe it never made it to the server?

However, as I say, I'm not used to potlatch so may have made mistakes.
